charts/tidb-cluster/values.yaml

```diff
@@ -56,7 +56,7 @@ discovery:
 enableConfigMapRollout: true
 
 # Whether enable TLS connections between server nodes.
-# When enabled, PD/TiDB/TiKV will use TLS encrypted connections to transfer data between each node,
+# When enabled, PD/TiDB/TiKV/PUMP will use TLS encrypted connections to transfer data between each node,
```

Comment:
We need to make it clear in the config section for pump and drainer that no need to config the security parameters for tls.